### ISSUE TITLE
Question about experiments on Stanford2D3D_Pan datasets

### ISSUE THREAD

**Issue by WuTao-CS** [jamycheung/Trans4PASS#5]
Hi~,after reading your work "Behind Every Domain There is a Shift: Adapting Distortion-aware Vision Transformers for Panoramic Semantic Segmentation", I have great admiration for your wonderful work. But when I tried to reproduce the supervised training results of Trans4Pass+ in Stanford2D3D_Pan, I encountered some problems. I tried to use the configuration in configs/stanford2d3d_pan/trans4pass_plus_small_1080x1080.yaml to conduct experiments on 4 NVIDIA V100 GPUs, and all parameters are unmodified. I have conducted three experiments on fold_1 and the results are 50.909, 51.299, and 51.092, which are quite different from the experimental results of 53.62 obtained in Table 8c in your paper.

The following is my training log file, I want to know where I have a problem with the use, and can you provide the original log file of your training?

[2022-11-25-20-12_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt](https://github.com/jamycheung/Trans4PASS/files/10095703/2022-11-25-20-12_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt)
[2022-11-22-16-46_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt](https://github.com/jamycheung/Trans4PASS/files/10095704/2022-11-22-16-46_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt)
[2022-11-25-20-11_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt](https://github.com/jamycheung/Trans4PASS/files/10095705/2022-11-25-20-11_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt)

Thank you!

**Comment by jamycheung** [comment]
Hi, 

thanks for your interest and question. We have checked the config file of the supervised training on the Stanford2D3D_Pan dataset. Since the dataset has a limited scale of data, we train our model with 100 epochs, and the batch size is set as 4. The learning rate is 0.0001. The evaluation size is 2048x1024. We have updated the relevant [config file](configs/stanford2d3d_pan/trans4pass_plus_small_1080x1080.yaml). We post our training log file below.

[2022-06-17-02-28_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt](https://github.com/jamycheung/Trans4PASS/files/10098453/2022-06-17-02-28_Trans4PASS_trans4pass_v2_stanford2d3d_pan_log.txt)

Thank you!
